Control/lib/middleware/addDetectorId.middleware.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Middleware function to add detectorID to the request object
+ * @return {function(req, res, next): void} - middleware function
+ */
+const addDetectorIdMiddleware = (detectorId) => {
+  return async (req, res, next) => {
+    req.params.detectorId = detectorId;
+    next();
+  };
+}
+exports.addDetectorIdMiddleware = addDetectorIdMiddleware
```

Comment:
Please do not forget to add a new line at the end of the file

Reply:
@pepijndik , you marked the conversation as resolved but there is still no end of line for the file